### ISSUE TITLE
The last property won't display.

### ISSUE THREAD

**Issue by ljvictorio** [gregwiechec/bulk-edit-documentation#2]
If a content or media type has only one property, the bulk edit modal will be blank. If there are two properties, the first one will show but the second one won't. If there are three or more properties, the last one will not display in the Bulk Edit modal. 
![BulkEdit1](https://github.com/gregwiechec/bulk-edit-documentation/assets/6375196/116224e3-ac49-4579-a607-e123bc6af44b)
![BulkEdit2](https://github.com/gregwiechec/bulk-edit-documentation/assets/6375196/8a034ae9-5cba-439f-9f7e-35f30d71c1fd)

**Comment by gregwiechec** [comment]
I tried this locally and I have all properties visible:

`[ContentType(GUID = "4BCCE6A3-1271-4B48-A60C-685850E432BA", GroupName = Global.GroupNames.Specialized)]
public class TestPage: PageData
{
    public virtual string Property1{ get; set; }

    public virtual string Property2{ get; set; }

    public virtual string Property3{ get; set; }

    public virtual string Property4{ get; set; }

    public virtual string Property5{ get; set; }

}`

![image](https://github.com/gregwiechec/bulk-edit-documentation/assets/12434079/21ff1a39-93b0-4c11-8f2e-027399cb4e53)


When you copy my example, you also don't see `Property5`?

**Comment by ljvictorio** [comment]
I used your example locally and I do see property 5 like you when using Test Page : PageData as the base class. I do not see Category like you do however. 
![image](https://github.com/gregwiechec/bulk-edit-documentation/assets/6375196/27e287df-5b62-498b-b8bc-1e2803ccc336)
![image](https://github.com/gregwiechec/bulk-edit-documentation/assets/6375196/1f71437a-61d2-447a-9e91-663e6295bad9)

The last property or category is not displaying on my ImageFile class which inherits from ImageData base class.

**Comment by gregwiechec** [comment]
The category will be available in version 1.2.0. Let me know what you see with this version

**Comment by ljvictorio** [comment]
I do see the category now, that looks good. I left a comment in the other issue. 
I'm still not seeing the last property of the ImageFile class however. 
I'm not able to see thirdProperty when trying to bulk edit a few jpg files. Are you able to recreate this issue?
![image](https://github.com/gregwiechec/bulk-edit-documentation/assets/6375196/90bae769-0072-4fb9-8153-c496648ab6e3)
![image](https://github.com/gregwiechec/bulk-edit-documentation/assets/6375196/94408824-6570-42fe-a75f-6f6fc290a644)

**Comment by gregwiechec** [comment]
So the `TestPage` example is working for you? Do you see all properties in the form?

**Comment by ljvictorio** [comment]
Yes. I do see all properties in the form for the page types that I've tested.
I'm only able to recreate the missing property issue on block types and media types.

**Comment by gregwiechec** [comment]
Could you give me code snippet of your block or media so I can test it?

**Comment by ljvictorio** [comment]
Here is the imageFile.cs class. 

```
using EPiServer.Framework.DataAnnotations;
namespace AlloyDemo.Models.Media;
[ContentType(GUID = "0A89E464-56D4-449F-AEA8-2BF774AB8730")]
[MediaDescriptor(ExtensionString = "jpg,jpeg,jpe,ico,gif,bmp,png")]
public class ImageFile : ImageData
{
    /// <summary>
    /// Gets or sets the copyright.
    /// </summary>
    /// <value>
    /// The copyright.
    /// </value>
    public virtual string Copyright { get; set; }

    public virtual string firstProperty { get; set; }

    public virtual bool secondProperty { get; set; }

    public virtual int thirdProperty { get; set; }

}
```

**Comment by ljvictorio** [comment]
And here is my ConfigureServices method

```
public void ConfigureServices(IServiceCollection services)
    {
        if (_webHostingEnvironment.IsDevelopment())
        {
            AppDomain.CurrentDomain.SetData("DataDirectory", Path.Combine(_webHostingEnvironment.ContentRootPath, "App_Data"));

            services.Configure<SchedulerOptions>(options => options.Enabled = false);
        }

        services
            .AddCmsAspNetIdentity<ApplicationUser>()
            .AddCms()
            .AddFind()
            .AddAlloy()
            .AddAdminUserRegistration()
            .AddEmbeddedLocalization<Startup>()
            .AddBulkEdit();
        services.Configure<BulkEditUiOptions>(o =>
        {
            o.PageTreeCommandEnabled = true;
            o.ProjectCommandEnabled = true;
            o.MaximumSafeContentItemsToUpdate = 1000;
            o.AssetsPaneCommandEnabled = true;
        });
        // Required by Wangkanai.Detection
        services.AddDetection();

        services.AddSession(options =>
        {
            options.IdleTimeout = TimeSpan.FromSeconds(10);
            options.Cookie.HttpOnly = true;
            options.Cookie.IsEssential = true;
        });
        services.AddTinyMceCustomizations();
    }
```

**Comment by gregwiechec** [comment]
Thank you code snippet. I found the bug. It will be fixed in version 1.2.1
